CHANGES.md

```diff
@@ -2,13 +2,20 @@
 
 ## __NEXT__
 
+### Major Changes
+
+* `augur mask --mask`, `augur tree --exclude-sites`: BED files with inconsistent CHROM values (i.e., values in the first column of data lines) will throw an error, as Augur (implicitly) expects to be working on a single piece of DNA (chromosome, segment, etc), and multiple CHROM values in a BED file indicate a violation of this expectation. This is a breaking change. [#945][] (@genehack)
```

Comment:
(nit)

Previously it was implicit, but if Augur now throws an error, isn't it explicit?

```suggestion
* `augur mask --mask`, `augur tree --exclude-sites`: BED files with inconsistent CHROM values (i.e., values in the first column of data lines) will throw an error, as Augur expects to be working on a single piece of DNA (chromosome, segment, etc), and multiple CHROM values in a BED file indicate a violation of this expectation. This is a breaking change. [#945][] (@genehack)
```

Reply:
Unless there's a specific statement in the documentation where there's an unequivocal statement to that effect (in which case, the CHANGES file should be referencing that), then no, to me the assumption is still implicit.

Reply:
Fair. I guess the underlying reasoning behind all my comments regarding this new error is that I don't see the reason for adding it now, unless we make the motivation explicit and clear (e.g. with new documentation if that's what's needed for explicitness).